_data/simple-icons.json

```diff
@@ -6282,6 +6282,11 @@
                 "type": "GPL-3.0-or-later"
             }
         },
+        {
+            "title": "Kick",
+            "hex": "53FC19",
+            "source": "https://www.dropbox.com/sh/zzjhpjpzdiegwz9/AAC7FV6oLeHtb9eRruLQ-_p2a?dl=0"
```

Comment:
```suggestion
            "source": "https://kick.com"
```

Reply:
The dropbox link is officially mentioned by [kick](https://kick.com) in their brand section in footer. Do I still need to change them?

Reply:
Yep - we're much better placed to link to their website (which has the obvious 'brand' link) as opposed to a random Dropbox URL.